bundler/lib/bundler/cli/gem.rb

```diff
@@ -68,6 +68,7 @@ def run
         test: options[:test],
         ext: extension,
         exe: options[:exe],
+        bundle: options[:bundle],
```

Comment:
You need to add the new option as a `method_option` in `bundler/lib/bundler/cli.rb`. Otherwise this option is not recognized by the CLI.

Also, how do you feel about calling the new configuration `install`, instead?